mscore/inspector/inspectorImage.cpp

```diff
@@ -56,10 +62,42 @@ InspectorImage::InspectorImage(QWidget* parent)
 
 void InspectorImage::valueChanged(int idx)
       {
-      InspectorBase::valueChanged(idx);
+      InspectorElementBase::valueChanged(idx);
```

Comment:
Unrelated bug fix?

Reply:
Well, I saw that and thought this needed to be changed, so I changed it ;-)
Do you mean I should separate it from this commit? I don't see the need actually.

Reply:
One of the boxes that you're supposed to check when creating a PR affirms that you didn't make any unnecessary changes. I try my best to follow that, and I assumed that everyone else did too.

Reply:
This is a _necessary_ change to ensure the function of the correct parent class being called.

Reply:
I take “necessary” to mean in the context of what's being fixed in the PR. Unless I misread the code, the parent class isn't actually doing anything in this case, so this fix, while good, is not necessary for this PR.

I don't know how much of a stickler we're expected to be, but I personally err on the side of caution.

Reply:
I've always been doing these changes which, although irrelevant to the issue being resolved, are good and cannot possibly have any adverse impact. It seems a bit much to create another commit just for this one-line change, and if we don't change this now, it may lead to a bug later on when the functionality of `InspectorElementBase::valueChanged()` is expanded.

Reply:
Fair enough. Just thought I'd ask.